src/pal/src/configure.cmake

```diff
@@ -366,6 +366,9 @@ check_cxx_source_runs("
 #include <stdlib.h>
 #include <time.h>
 #include <sys/time.h>
+#ifdef (__linux__)
```

Comment:
> What do you mean by "the __LINUX__ is not valid"? 
Right. Please, refer to (Linux GCC Howto)".
* Section: porting and compiling 
http://www.faqs.org/docs/Linux-HOWTO/GCC-HOWTO.html
Anyway, I will try to build with _  _ LINUX _ _  that you commented.